data/scenarios/scenarios.json

```diff
@@ -0,0 +1,10 @@
+"scenarios": [
+    {
```

Comment:
Could we make use of, for example, an AMI ID here, in case we wanted starter boxes different than "generic base box"? For example, one with a broken Redis instance user would be expected to troubleshoot?